src/screens/TransactionViews/TransactionViewUtils.res

```diff
@@ -31,13 +35,26 @@ let getAllViewsString = obj => {
   ->Array.joinWith(",")
 }
 
-let getViewsString = (view, obj) => {
-  switch view {
-  | All => getAllViewsString(obj)
-  | Succeeded => "succeeded"
-  | Failed => "failed"
-  | Dropoffs => "requires_payment_method"
-  | Cancelled => "cancelled"
+let getViewsString = (view, obj, entity) => {
+  switch entity {
+  | Orders =>
+    switch view {
+    | All => getAllViewsString(obj)
+    | Succeeded => "succeeded"
+    | Failed => "failed"
+    | Dropoffs => "requires_payment_method"
+    | Cancelled => "cancelled"
+    | Pending => "pending"
+    }
+  | Refunds =>
+    switch view {
+    | All => getAllViewsString(obj)
+    | Succeeded => "success"
+    | Failed => "failure"
+    | Pending => "pending"
```

Comment:
Why `Succeeded` , `Failed` values are different for orders and refunds . Shouldn't it  be consistent ?

Reply:
Is this what backend is expecting ? if yes then change the function name to something more intuitive

Reply:
string vales are different because this is how backend is expecting

Reply:
i think the name is adhering to convention, since we are getting the string for respective views using this function